.github/workflows/config.yaml

```
@@ -0,0 +1,32 @@
+  push:
+    branches: [type]  # Run mypy on each push to main
+  pull_request:
+    branches: [type]  # Run mypy on each PR to main
+
+jobs:
+  mypy-check:
+    runs-on: windows-latest  # Specify your OS (windows-latest, ubuntu-latest, etc.)
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)